website/content/docs/concepts/seal.mdx

```diff
@@ -8,145 +8,130 @@ description: >-
 
 # Seal/Unseal
 
-When a Vault server is started, it starts in a _sealed_ state. In this
-state, Vault is configured to know where and how to access the physical
-storage, but doesn't know how to decrypt any of it.
+When you start a Vault server, it starts in a sealed state. In this
+state, Vault can access the physical
+storage, but it cannot decrypt any of the data on it.
 
-_Unsealing_ is the process of obtaining the plaintext root key necessary to
-read the decryption key to decrypt the data, allowing access to the Vault.
+Unsealing is the process of obtaining the plaintext root key that is necessary to
+read the decryption key. The decryption key allows you to decrypt the data and access the Vault.
```

Comment:
Suggest we remove the italics for emphasis. Italics are an accessibility issue. We can use bold if necessary, but I don't think we particularly need emphasis here.
Also, passive voice.